packages/round-manager/src/features/round/RoundApplicationForm.tsx

```diff
@@ -342,6 +362,42 @@ function ApplicationInformation(props: {
         className="inline h-4 w-4 mb-1 ml-2 text-grey-400"
         data-testid={"edit-title"}
       />
+
+      {/* Encrypted Toggle */}
+      <Switch.Group as="div" className="flex items-center justify-between">
+        <span className="flex flex-grow flex-col">
+          <Switch.Label as="span" className="text-sm font-medium text-gray-900" passive>
+            <p className="text-xs">Encrypted</p>
+          </Switch.Label>
+        </span>
+        <Switch
+          {...register(`applicationMetadata.questions.${index}.encrypted`)}
+          checked={encryptedStates[index]}
+          onChange={() => {
+
+            const newEncryptedState = [...encryptedStates];
+            newEncryptedState[index] = !encryptedStates[index];
+            setEncryptedStates(newEncryptedState);
+
+            // THIS IS BROKEN
+            // WHERE TO REGISTER
```

Comment:
@shavinac this seems to fail and my hunch is cause the registering doesn't happen right

Reply:
i think this is fixed by using a [form controller](https://react-hook-form.com/api/usecontroller/controller) instead of local state to control the switch value. and also manually setting `checked` [prop on the Switch](https://headlessui.com/react/switch#switch)
```
<Switch
    {...field}
    checked={field.value}
    value={field.value.toString()}
    className={classNames(
      field.value ? "bg-indigo-600" : "bg-gray-200",
      "relative inline-flex h-6 w-11 flex-shrink-0 cursor-pointer rounded-full border-2 border-transparent transition-colors duration-200 ease-in-out focus:outline-none focus:ring-2 focus:ring-indigo-500 focus:ring-offset-2"
    )}
    data-testid="encrypted-toggle"
>
```

had to set the `value` prop to override the `value` set by the `{...field}`. otherwise the TS compiler complains that the `field.value` (boolean) and Switch's `value` (string/number/undefined) don't match 😒 😩